Tools/Scripts/libraries/webkitbugspy/webkitbugspy/mocks/base.py

```diff
@@ -56,4 +56,6 @@ def add(self, bug_data):
             if user:
                 self.users.add(type(self).transform_user(user))
 
-        self.issues[id] = {key: value for key, value in bug_data.items()}
+        self.issues[id] = {}
+        for key, value in bug_data.items():
+            self.issues[id][key] = [e for e in value] if isinstance(value, (list, tuple)) else value
```

Comment:
Should we use deep copy instead or we believe we will only have two level nested in `bug_data`?

Reply:
Forgot about `copy.deepcopy`! That's definitely the right approach.